### ISSUE TITLE
Compatibility with gradle 7.0

### ISSUE THREAD

**Issue by cocorossello** [nilsmagnus/wsdl2java#128]
Compile scope is removed in gradle 7.0, see https://github.com/nilsmagnus/wsdl2java/pull/127

**Comment by rpost** [comment]
Any news on that? Gradle 7.0 is available now.

**Comment by cvetan** [comment]
Also interested in this. Because gradle 7 is natively supporting new apple silicon architecture, and we really need this.

**Comment by rpost** [comment]
Worth to share it here: https://github.com/nilsmagnus/wsdl2java/commit/a9b54484f5bb7b60f9932f6e6bf488ce7c51bf1d

**Comment by isole** [comment]
Just use this [one](https://github.com/yupzip/wsdl2java). Works right out of the box.

**Comment by Deepu978** [comment]
any update on this ?

**Comment by h908714124** [comment]
@isole Is your forked version available yet on maven central, or some other standard repo?

**Comment by isole** [comment]
Yup, [right here](https://mvnrepository.com/artifact/com.yupzip.wsdl2java/com.yupzip.wsdl2java.gradle.plugin). We are using version 2.0 in production. Simply add the following in your plugins section:

`id "com.yupzip.wsdl2java" version "2.0"`

**Comment by h908714124** [comment]
Can you reach out to the yupzip maintainers and ask them to enable issues on their wsdl2java repo?
